ui/app/routes/evaluations/AdvancedParametersAccordion.tsx

```diff
@@ -0,0 +1,67 @@
+import { useState } from "react";
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "~/components/ui/accordion";
+import { Label } from "~/components/ui/label";
+import { RadioGroup, RadioGroupItem } from "~/components/ui/radio-group";
+import type { InferenceCacheSetting } from "~/utils/evaluations.server";
+
+export interface AdvancedParametersAccordionProps {
+  inference_cache: InferenceCacheSetting;
+  setInferenceCache: (inference_cache: InferenceCacheSetting) => void;
+}
+
+export function AdvancedParametersAccordion({
+  inference_cache,
+  setInferenceCache,
+}: AdvancedParametersAccordionProps) {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <Accordion
+      type="single"
+      collapsible
+      className="w-full"
+      value={isOpen ? "advanced-parameters" : undefined}
+      onValueChange={(value) => setIsOpen(value === "advanced-parameters")}
+    >
+      <AccordionItem value="advanced-parameters">
+        <AccordionTrigger className="hover:no-underline">
+          <div className="flex items-center gap-1">
+            <span>Advanced Parameters</span>
+          </div>
+        </AccordionTrigger>
+        <AccordionContent>
+          <div className="space-y-6 px-3 pt-3">
+            <Label>Caching</Label>
+            <RadioGroup
+              value={inference_cache}
+              onValueChange={setInferenceCache}
+              className="mt-2 flex gap-4"
+            >
+              <div className="flex items-center space-x-2">
+                <RadioGroupItem value="on" id={`on`} />
```

Comment:
I think you can just do `id="on"` no?

Reply:
(same below)

Reply:
don't think this is true, the [reference](https://ui.shadcn.com/docs/components/radio-group) shows both in there